README.md

```diff
@@ -30,6 +30,37 @@ Note: If dns servers specified on the host or via dns flags are unreachable from
 
 - https://docs.docker.com/config/containers/container-networking/#dns-services
 
+### docker.login
+
+Authenticate the system user `root` against a set of custom docker registries.
+The registries and the respectively required credentials should be provided through pillar data.
+This state primarily makes use of the `dockermod` salt module [documented here](https://docs.saltproject.io/en/latest/ref/modules/all/salt.modules.dockermod.html).
+See `pillar.example` for further details.
+
+### docker.machine
+
+Substate to install `docker-machine`.
+Since docker-machine is deprecated upstream, this state relies on the fork maintained by Gitlab.
+See [releases](https://gitlab.com/gitlab-org/ci-cd/docker-machine/-/releases) for available version strings for the `docker-machine:version` pillar key.
+
+When docker-machine is executed for the first time, a set of keys and certificates are generated.
+**The contents of these files must be manually generated and provided through pillar data.**
+To generate these files, simply run `docker-machine create --driver <DRIVER> <OPTIONS> init-vm` on a pristine machine, i.e. one where docker-machine is installed but was never invoked.
```

Comment:
I think the command in this line can be added to `pillar.example`, as a tl;dr version?